### DISCUSSION TITLE
Is it possible to have just one location for the R code (ex : libraries, reading data) which concerns multiple pages of my webiste ?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#8182] [Q&A] [answered]
### Description

Is it possible to have just one location for the R code (ex : libraries, reading data) which concerns multiple pages of my webiste ?
I feel like I have to repeat the same bock code calling libraries and data on every page.

**Comment by cderv** [comment, marked as answer]
Not sure if this was closed by accident but you have several solutions

- Use Quarto feature  includes: https://quarto.org/docs/authoring/includes.html to reuse some content
- Load R script in a chunk. Example for knitr (https://bookdown.org/yihui/rmarkdown-cookbook/option-code.html) - you can adapt the option to use YAML
- Use knitr child document (https://bookdown.org/yihui/rmarkdown-cookbook/child-document.html) - useful for more complex content to share that requires knitting

Hope it helps
